runtime/doc/usr_05.txt

```diff
@@ -250,121 +250,115 @@ The ":map" command (with no arguments) lists your current mappings.  At
-*05.4*	Adding a plugin					*add-plugin* *plugin*
-
-Vim's functionality can be extended by adding plugins.  A plugin is nothing
-more than a Vim script file that is loaded automatically when Vim starts.  You
-can add a plugin very easily by dropping it in your plugin directory.
-{not available when Vim was compiled without the |+eval| feature}
-
-There are two types of plugins:
-
-    global plugin: Used for all kinds of files
-  filetype plugin: Only used for a specific type of file
-
-The global plugins will be discussed first, then the filetype ones
-|add-filetype-plugin|.
-
-
-GLOBAL PLUGINS						*standard-plugin*
-
+*05.4*	Plugins					*add-plugin* *plugin*
+
+Vim's functionality can be extended by adding plugins.  A plugin is
+essentially a set of Vim script files that are loaded automatically when Vim
+starts or on some other event (like opening a file of a certain type).
+Plugins can be used to add support to more programming languages, extend Vim's
+command "vocabulary", simplify some use-cases, add some bling to Vim, and even
+turn it into a different sort of application.  Thankfully, the community has
+made available on the net a great ammount of plugins that any user can benefit
+from.
+
+You can add a plugin very easily by dropping it in your |vimfiles| directory,
+but there are more advanced ways to manage plugins, which will be described
+below.
+
+KINDS OF PLUGIN                                         *plugin-types*
+
+There are mainly two kinds of plugin: global and filetype-specific.
+
+Global plugins are sourced on startup and usually add functionality that will
+be available regardless of the opened files.  For example, the |matchparen|
+plugin can be used in any kind of file whatsoever to highlight matching
+parentheses on cursor movement.
+					                *standard-plugin*
 When you start Vim, it will automatically load a number of global plugins.
 You don't have to do anything for this.  They add functionality that most
 people will want to use, but which was implemented as a Vim script instead of
 being compiled into Vim.  You can find them listed in the help index
 |standard-plugin-list|.  Also see |load-plugins|.
 
-							*add-global-plugin*
-You can add a global plugin to add functionality that will always be present
-when you use Vim.  There are only two steps for adding a global plugin:
-1. Get a copy of the plugin.
-2. Drop it in the right directory.
+Global plugins reside under the `plugin/` |vimfiles| directory.
+
+                                                        *ftplugins*
+Filetype plugins ("ftplugins") are only used for a specific type of file, and
+usually change the buffer-local settings.  For example, the standard vim C
+ftplugin sets 'formatoptions' to break comment lines but not other lines, and
+to insert the comment leader when hitting <CR> or using |o|.  Filetype plugins
+are usually distributed alongside syntax files, and indent and filetype
```

Comment:
I don't like the double and here, as in "...usually distributed alongside syntax files, *and* indent *and* filetype detection scripts..." but I am not sure how to fix it.